### PR TITLE
Yarn to npm

### PR DESCRIPTION
**Pull request by abelalmeida**
# Description                                                                   
                                                                                   
  I was attempting to install yarn 1.22.5, but only yarn version 3 would install. While on the call with @mattwynne he suggested we run our test using `npm run` which led to changing `yarn` to `npm run` in CONTRIBUTING.md.
                                                                                   
  # Motivation & context                                                          
                                                                                  
  _Why is this change required? What problem does it solve?                       
  If it fixes an open issue, please link to the issue here._                      
  Suggestion from Matt Wynne, that the team has been trying to run tests using npm run instead of yarn.
                                                                                                                                                                       
  ## Type of change                                                               
                                                                                   
                                
                                                                                   
  - Refactoring/debt (improvement to code design or tooling without changing behaviour)
   Now instead of using yarn we will be using npm to run the testing in the project.
                                                                                  
  _Remember to add an entry to the relevant section of CHANGELOG.md as part of this pull request._
                                                                                          
  # Checklist:                                                                    
                                                                                   
  
  
                                                                                   
  - [ ] My code follows the code style of this project.                           
  - [X] My change requires a change to the documentation.                         
  - [X] I have updated the documentation accordingly.                             
  - [X] I have read the [**CONTRIBUTING**](../CONTRIBUTING.md) document.          
  - [ ] I have added tests to cover my changes.                                   
  - [X] All new and existing tests passed.

### CODE REVIEW

**Comment by mattwynne** [comment]
Thanks @abelalmeida, it was really confusing when you couldn't run the tests because following the instructions in `CONTRIBUTING.md` had led you to install yarn 3, which didn't work.

Any objections @davidjgoss, or anything we missed??

**Comment by mattwynne** [comment]
@abelalmeida I've noticed one thing we missed: https://github.com/cucumber/cucumber-js/blob/main/.github/workflows/build.yml

Can you update that to use `npm ` too?

**Comment by abelalmeida** [comment]
> @abelalmeida I've noticed one thing we missed: https://github.com/cucumber/cucumber-js/blob/main/.github/workflows/build.yml
> 
> Can you update that to use `npm` too?

@mattwynne  I just adjusted the build.yml, please have a look 
thanks, Abel

File: package.json [hunk 1]
```diff
@@ -259,19 +259,19 @@
...
-    "pretypes-test": "yarn build-local",
+    "pretest-coverage": "npm run build-local",
+    "pretypes-test": "npm run build-local",
     "test-coverage": "nyc --silent mocha 'src/**/*_spec.ts' 'compatibility/**/*_spec.ts' && nyc --silent --no-clean node ./bin/cucumber-js && nyc report --reporter=lcov",
-    "test": "yarn run lint && yarn run types-test && yarn run unit-test && yarn run cck-test && yarn run feature-test",
+    "test": "npm run lint && npm run types-test && npm run unit-test && npm run cck-test && npm run feature-test",
     "types-test": "tsd",
     "unit-test": "mocha 'src/**/*_spec.ts'",
-    "update-dependencies": "npx npm-check-updates --upgrade"
+    "update-dependencies": "npx npm run-check-updates --upgrade"
```
Comment (davidjgoss) [inline package.json:274]: This doesn't look quite right - does it work?

**Review by davidjgoss — COMMENTED** [review]
Looks good, just a small query, thanks.

File: package.json (hunk 1 above)
Comment (abelalmeida) [reply package.json]: @davidjgoss, test the change did not work so I reverted back the changes.I should have the individual commands but did not so sorry. 
Abel

Comment (davidjgoss) [reply package.json]: No problem, thanks!

**Review by davidjgoss — APPROVED** [review]
Got this green by ensuring we install npm 7 for every Node version we build with, so it knows how to deal with the newer lockfile format.

Thanks for doing this!

**Comment by aslakhellesoy** [comment]
Hi @abelalmeida,

Thanks for your making your first contribution to Cucumber, and welcome to the Cucumber committers team! You can now push directly to this repo and all other repos under the cucumber organization! 🍾

In return for this generous offer we hope you will:

* ✅ Continue to use branches and pull requests. When someone on the core team approves a pull request (yours or someone else's), you're welcome to merge it yourself.
* 💚 Commit to setting a good example by following and upholding our [code of conduct](https://cucumber.io/conduct) in your interactions with other collaborators and users.
* 💬 Join the [community Slack channel](http://cucumber.io/support#slack) to meet the rest of the team and make yourself at home.
* ℹ️ Don't feel obliged to help, just do what you can if you have the time and the energy.
* 🙋 Ask if you need anything. We're looking for feedback about how to make the project more welcoming, so please tell us!

On behalf of the Cucumber core team,
Aslak Hellesøy
Creator of Cucumber

**Comment by mattwynne** [comment]
\o/ Congrats @abelalmeida! 💖 

What's next? Ping me or book me at https://calendly.com/mattwynne if you want another pairing session to work on your next contribution. 💥

**Pull request merged**